docs/scripts/util/crawlAllLinksFromAllPages.ts

```diff
@@ -0,0 +1,41 @@
+import puppeteer from 'puppeteer';
+import { runArrayPromiseInOrder } from '.';
+
+export async function crawlAllLinksFromAllPages(allPages: string[]) {
```

Comment:
This name feels extraneously verbose, can it just be:
```
function crawlAllLinks(pages: string[]) {}
```

Reply:
Done 6554210ee91d644d5279c7414a0cbf0a6764dc75